src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    public static final int OPERATION_TYPE = 0;
+    public static final int AMOUNT = 1;
+
+    public void getStatistic(String fromFileName, String toFileName) throws IOException {
+        int supplyCounter = 0;
+        int buyCounter = 0;
+
+        try (FileReader fileReader = new FileReader(fromFileName)) {
```

Comment:
Use try-with-resources to ensure that the `BufferedReader` instance is also closed after use, not just the `FileReader`.